platform-configs/include/default.robot

```diff
@@ -63,6 +63,7 @@ ${DEVICE_AUDIO1_WIN}=                               ${EMPTY}
 ${WIFI_CARD_UBUNTU}=                                ${EMPTY}
 ${USB_MODEL}=                                       ${EMPTY}
 ${USB_DEVICE}=                                      ${EMPTY}
+@{ATTACHED_USB}=                                    ${EMPTY}
```

Comment:
We need to include boot device names in the config for each board?

Reply:
I would say yes, all global variables should probably be in some default state instead of fixing errors that no keyword exists.
In this case this line from [usb-detect.robot](https://github.com/Dasharo/open-source-firmware-validation/blob/a6145514ee111e264aef873551b2bbc473b9626a/dasharo-compatibility/usb-detect.robot#L52) would always fail for all protectli except `vp2410` and `vp2420`.

Do we even need this variable? in `raptor-cs_talos2.robot` config there is comment:
`# Temporary parameter - we are not able to use the Hardware matrix`
If this problem was fixed then we could just get rid of it as it was only used in `usb-detect.robot`